integration-tests/src/test/java/org/openmrs/maven/plugins/CreateProjectIntegrationTest.java

```diff
@@ -0,0 +1,64 @@
+package org.openmrs.maven.plugins;
+
+import org.apache.maven.model.Model;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+
+public class CreateProjectIntegrationTest extends AbstractSdkIntegrationTest {
+
+
+    Model model;
+
+    List<String> options = Arrays.asList("moduleId", "moduleName", "version", "groupId", "artifactId", "moduleDescription");
+
+    @Before
+    public void setup() throws Exception {
+        //test pom needs reset after this test, because archetype plugin modifies it
+        //overwrite superclass setup to ensure testDirectory is initialized
+        super.setup();
+        model = getTestPom(testDirectory);
+    }
+
+    @After
+    public void resetPom() throws Exception {
+        saveTestPom(testDirectory, model);
+    }
+
+    @Test
+    public void createProject_shouldCreateRefappModuleProject() throws Exception{
+        addTaskParam("moduleType", "refapp");
```

Comment:
Could we rename it to just type? I envision create-project being called for OWAs, creating distributions, etc. We'll have the following types to start with: platform-module, referenceapplication-module